grapple/types/images.py

```diff
@@ -22,7 +27,25 @@ class BaseImageObjectType(graphene.ObjectType):
     url = graphene.String(required=True)
     aspect_ratio = graphene.Float(required=True)
     sizes = graphene.String(required=True)
+
     collection = graphene.Field(lambda: CollectionObjectType, required=True)
+    placeholder_blur = graphene.String()
+
+    def resolve_placeholder_blur(self, info, **kwargs):
+        """
+        Get a low quality image placeholder of the original image.
+        This can directly be use in frontend frameworks like next.js
+        """
+        rendition = self.get_rendition("width-64|jpegquality-50")
```

Comment:
question: could the quality go lower?

Reply:
It could, we could even use smaller image in size. 
I did not use this behaviour a lot, it might requires some tweaking in the long run. 

I can provide a few examples first to compare various sizes/qualities. Let me know what you prefer